bitnami/postgresql/templates/svc-read.yaml

```diff
@@ -8,15 +8,17 @@ metadata:
     chart: {{ template "postgresql.chart" . }}
     release: {{ .Release.Name | quote }}
     heritage: {{ .Release.Service | quote }}
-{{- with .Values.service.annotations }}
-  annotations:
-{{ toYaml . | indent 4 }}
-{{- end }}
+  {{- if .Values.service.annotations }}
```

Comment:
Could we add sth in documentation to understand they are evaluated as a template?